wherehows-web/app/routes/application.js

```diff
@@ -31,6 +32,19 @@ export default Route.extend(ApplicationRouteMixin, {
   },
 
   /**
+   * __It reloads the Ember.js application__ by redirecting the browser
+   * to the application's root URL so that all in-memory data (such as Ember
+   * Data stores etc.) gets cleared.
+   * @override ApplicationRouteMixin.sessionInvalidated
+   * @link https://github.com/simplabs/ember-simple-auth/issues/1048
+   */
+  sessionInvalidated() {
+    if (!testing) {
+      window.location.replace('/');
```

Comment:
I think the point of a `location.replace` is to *__force__* the browser to clear it's cache. If you're using ESA (Ember Simple Auth) on TREX, this is what it is doing by default when the user clicks logout in the browser. 

From ESA: https://github.com/simplabs/ember-simple-auth/blob/master/addon/mixins/application-route-mixin.js#L135

```javascript

      if (this.get('_isFastBoot')) {
        this.transitionTo(Configuration.baseURL);
      } else {
        window.location.replace(Configuration.baseURL);
      }
```
If you use `.transitionTo` or `.transitionTo('login')`, then the SPA will retain *__all__* the the previous information in it's cache which is a bug and security issue.

For example:, if I logout and a new user logs in, then if the app is using Ember.Data or ember-redux to manage state, all those in-memory objects will not be cleared. Therefore, the newly logged in user will see all the data that was retained from the previous user's session. By forcing the browser to request those resources, you ensure the you're starting with a clean slate. Hope this clears things up?
@justinpark  @justYves